lighthouse-core/test/gather/computed/computed-artifact-test.js

```diff
@@ -44,4 +65,23 @@ describe('ComputedArtifact base class', () => {
       assert.equal(result, 1);
     });
   });
+
+  it('caches multiple input arguments', () => {
+    const mockComputed = {computed: true};
+    const computedArtifact = new MultipleInputArtifact(mockComputed);
+
+    const obj0 = {value: 1};
+    const obj1 = {value: 2};
+    const obj2 = {value: 3};
+
+    return computedArtifact.request(obj0, obj1)
+      .then(result => assert.equal(result, 0))
+      .then(() => assert.deepEqual(computedArtifact.lastArguments, [obj0, obj1, mockComputed]))
```

Comment:
does testing `lastArguments` add more information? It seems like testing `result` is sufficient to check that a cached result is being returned.